### PR TITLE
Az oauth, add instructions for token claims and api permission.

### PR DESCRIPTION
**Pull request by hterik**
Without these steps the integration does not work.

Honestly I'm not 100% sure about the exact claims and permissions that are really needed for Grafana, these were mostly found by looking at logs and trial and error, after following the rest of the instructions on the changed page. Would be good if someone with deeper knowledge in this area could review.

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
Need dev review and testing.

**Comment by andresmgot** [comment]
not sure if our team is the code owner for Azure authentication but adding it to our kanban board to investigate it.

**Comment by achatterjee-grafana** [comment]
@andresmgot, any updates on this? Thank you :)

File: docs/sources/auth/azuread.md [hunk 1]
```diff
@@ -89,6 +89,13 @@ To enable the Azure AD OAuth2, register your application with Azure AD.
 
 1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
 
+1. Under Azure AD / App Registrations / Grafana / **Token Configuration**, click **Add Optional Claim** and select Type ***ID** and Claim **email**.
+
+1. Under Azure AD / App Registrations / Grafana / **API Permissions**, Add following permissions:
+    * **Directory.AccessAsUser.All** from **Azure AD Directory Graph**
```
Comment (alexanderzobnin) [inline docs/sources/auth/azuread.md:95]: Should it be `Delegated permissions` or `Application permissions` (guess, delegated)? Also, Azure AD Directory Graph is deprecated, and it's suggested to use Microsoft Graph APIs.

Comment (alexanderzobnin) [reply docs/sources/auth/azuread.md:95]: Screenshot from 2022-02-07 11-59-55

Screenshot from 2022-02-07 12-00-23

**Pull request closed without merging**